packages/run-protocol/src/vaultFactory/params.js

```diff
@@ -45,6 +48,10 @@ const makeVaultDirectorParams = (
       type: ParamTypes.UNKNOWN,
       value: liquidationTerms,
     },
+    [MIN_INITIAL_LOAN_KEY]: {
+      type: ParamTypes.AMOUNT,
```

Comment:
Is there any risk in this being a NAT? The brand is implied.

Reply:
It could be a NAT, but I like the clarity of specifying it in RUN. There was discussion in the issue of whether the constraint was on the collateral amount or the loan amount, and specifying it in RUN makes it clear that it's the latter.

Reply:
The debt brand is available here, so I think having this be an Amount is the right thing.